src/helpers/api.ts

```diff
@@ -258,25 +258,29 @@ export const fetchEachJob = async (
   return json;
 };
 
-export const fetchJobEvents = async (
-  accessToken: string | undefined,
-  jobId: any
-) => {
-  if (!accessToken || accessToken === undefined) {
-    redirect();
-    return;
-  }
-  const res = await fetch(`${url("/jobs")}/${jobId}/events`, {
-    next: {
-      tags: ["AllEvents"],
-    },
-    headers: {
-      Authorization: `Bearer ${accessToken}`,
-    },
+export const fetchJobEvents = async (jobId: any) => {
+  return apiCall(
+    `/events?q%5BfilterBy%5D[job][id][eq]=${jobId}&q%5BfilterBy%5D[job][id][eq]=${jobId}`
```

Comment:
not sure if this is correct. please can you test this once? it might work but just wanted to confirm